plugin/pkg/auth/authorizer/webhook/webhook.go

```diff
@@ -164,26 +163,26 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (err error) {
 		if err := result.Error(); err != nil {
```

Comment:
I actually think the 401 and 403 response codes are getting handled here... otherwise I don't see how the unit test that expects an error in 401/403 cases could be passing. Can you verify that, and remove the 401/403 handling below?